sympy/tensor/array/expressions/conv_array_to_indexed.py

```diff
@@ -1,84 +1,12 @@
-import collections.abc
-import operator
-from itertools import accumulate
+from sympy.tensor.array.expressions import from_array_to_indexed
+from sympy.utilities.decorator import deprecated
 
-from sympy import Mul, Sum, Dummy, Add
-from sympy.tensor.array.expressions import PermuteDims, ArrayAdd, ArrayElementwiseApplyFunc, Reshape
-from sympy.tensor.array.expressions.array_expressions import ArrayTensorProduct, get_rank, ArrayContraction, \
-    ArrayDiagonal, get_shape, _get_array_element_or_slice, _ArrayExpr
-from sympy.tensor.array.expressions.utils import _apply_permutation_to_list
 
+_conv_to_from_decorator = deprecated(
+    "module has been renamed by replacing 'conv_' with 'from_' in its name",
+    deprecated_since_version="1.11",
+    active_deprecations_target="deprecated-conv-array-expr-module-names",
```

Comment:
I think it would be better to use the `deprecated` decorator directly. I don't think DRY really matters here. It's more important that you can see this information when you look in each of these modules.